AtomicLib/Config.cs

```diff
@@ -0,0 +1,64 @@
+﻿using System.Xml.Serialization;
+using AtomicLib.XML;
+
+namespace AtomicLib;
+
+[Serializable]
+[XmlRoot("message")]
+public class Config
+{
+    public static string ConfigFilePath = Path.Combine(Environment.GetFolderPath(Environment.SpecialFolder.ApplicationData), "Atomic/config.xml");
+    
+    [XmlAttribute("Language")]
+    public string Language
+    { get; set; }
+    
+    [XmlAttribute("FillInContents")]
+    public bool FillInContents
+    { get; set; }
+    
+    [XmlElement("Fields")]
+    public FieldContents Fields
+    { get; set; }
+    
+    public Config()
+    {
+
+    }
+
+    public Config(string language, bool fillIn)
+    {
+        Language = language;
+        FillInContents = fillIn;
+        Fields = new FieldContents();
+    }
+    
+    public static Config LoadAndReturnConfig()
+    {
+        Config config = null;
+        if (File.Exists(ConfigFilePath))
+        {
+            string configXml = File.ReadAllText(ConfigFilePath);
+            config = Serializer.Deserialize<Config>(configXml);
+        }
+        else
+        {
+            config = CreateAndReturnDefaultConfig();
+        }
+        return config;
```

Comment:
I feel like
```suggestion
        if (!File.Exists(ConfigFilePath))
            return CreateAndReturnDefaultConfig(); 
                    
         string configXml = File.ReadAllText(ConfigFilePath);
         return Serializer.Deserialize<Config>(configXml);        
```

Is cleaner